CONTRIBUTING.md

```diff
@@ -26,5 +26,8 @@ You can find more [details on the project here](https://github.com/Junior-Devs/2
 
 We have a [Figma workspace](https://www.figma.com/file/g09N7BPg0QPBAKsr789UPo/Design-playground?node-id=0%3A1) to work on the mockups/design. Find more details on [how to add your ideas to the design here](https://github.com/Junior-Devs/2020-Junior-Devs-Project/wiki/How-to-collaborate-on-Figma).
 
-## Upcoming 
-Once we're happy with the plan and design we'll start creating tickets (GitHub issues) so we can start the work. Stay tuned!
+## How can you contribute?
+Tickets are created under [Issues](https://github.com/Junior-Devs/2020-Junior-Devs-Project/issues) tab in GitHub repository. 
+1) make sure you're registered with the group through e-mail and telegram first
+2) find issue that you can help with
+3) show your interest by starting conversation through GitHub issue commenting feature.
```

Comment:
Might be clearer to write "add a comment to the GitHub issue"

Reply:
Also, let's capitalize and add articles: "start *a* conversation" rather than "start conversation".

Reply:
ok let's go with shorter suggestion and I'll fix missing articles as well

Reply:
Now that I am trying to fix this, I am also thinking that maybe we should have a point:
4) to discuss implementation details, use #contributions channel on [Slack](link here)
What do you think?

Reply:
Also, while addressing all the points, I've realised that in CONTRIBUTING.md `General guidelines`, there is a list of steps which uses slightly different Markdown markup to the one I proposed in the section `How can you contribute?` I think it would be wise to select one approach and stick to it. Therefore the question is, should I change markup in `How can you contribute?` section to match it with `General guidelines`?

Reply:
While setting my local environment yesterday, I realised that info about setting local env in CONTRIBUTING.md was not accurate. I ended up copying steps we had in README.md as these do work and are accurate as of now. I used a nested list structure. Have a look through the file, let me know if you see any changes needed further.